lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -4,6 +4,9 @@
 
 @mixin focusable-toggle-input($focus-border-selector) {
   .nx-radio__input, .nx-checkbox__input {
+    position: absolute;
+    z-index: -1;
+
```

Comment:
Both of these attributes were removed (by me) because they didn't seem to do anything, it turns out they are necessary because FF ignores the `width: 0` attribute.